packages/protocol/contracts/layer1/prover-market/ProverMarket.sol

```diff
@@ -86,6 +86,12 @@ contract ProverMarket is EssentialContract, IProverMarket {
         minExitDelay = _minExitDelay;
     }
 
+    /// @notice Initializes the ProverMarket contract
+    /// @param _addressManager The address of the address manager contract
+    function initialize(address _addressManager) external initializer {
+        __Essential_init(_addressManager);
```

Comment:
`__Essential_init` accepts the owner address as its parmeter.

We should not use `_addressManager`, now we use `_resolver`, and it needs to be initialized in the constructor.

Reply:
See Bridge.sol as an example.